src/com/epam/javatrainings/annotation/validator/Validator.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.javatrainings.annotation.validator;
+import com.epam.javatrainings.annotation.model.CustomerDto;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Validator {
+  static Map<String, String> error = new HashMap<>();
```

Comment:
what will contain the error data if I would call the typeDetect method twice,
one time with a customer with an invalid name, and one with an invalid discountRate???